WebService 2 Assignment/WebServiceUserManagement/src/com/metacube/restwebservice/dao/EmployeeDao.java

```diff
@@ -0,0 +1,95 @@
+package com.metacube.restwebservice.dao;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.validation.constraints.Null;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.ParseException;
+
+import com.metacube.restwebservice.model.Employee;
+import com.metacube.restwebservice.util.Constant;
+import com.metacube.restwebservice.util.JsonFilehandling;
+
+/**
+ * This dao class is used to access the json file
+ * 
+ * @author Udit Saxena
+ *
+ */
+public class EmployeeDao {
+	private List<JSONObject> employeesList;
```

Comment:
Instead of keeping the list as a class variable it can be fetched and give the data to respective calls.